docker/docker-compose.yml

```diff
@@ -173,6 +175,36 @@ services:
       - "traefik.frontend.rule=HostRegexp:${COMPOSE_PROJECT_NAME:-default}.altis.dev;PathPrefix:/mailhog"
     environment:
       MH_UI_WEB_PATH: mailhog
+  analytics:
+    image: humanmade/local-pinpoint
+    ports:
+      - "3000"
+    networks:
+      - proxy
+      - default
+    labels:
+      - "traefik.port=3000"
+      - "traefik.protocol=http"
+      - "traefik.docker.network=proxy"
+      - "traefik.frontend.rule=HostRegexp:analytics-${COMPOSE_PROJECT_NAME:-default}.altis.dev"
+    restart: on-failure
+  kibana:
+    image: blacktop/kibana:6.3
```

Comment:
Is this an official image, could we get / use the elastic one here?

Reply:
It's the OSS version of KIbana. We're using `blacktop/elasticsearch` for the ES service so didn't think it was an issue. To use the official image requires building an image that uninstalls x-pack which this one doesn't have.